packages/ogmios/test/ogmiosToCore/__snapshots__/block.test.ts.snap

```diff
@@ -288,7 +288,7 @@ Object {
               "cbor": "d87980",
               "constructor": 0n,
               "fields": Object {
-                "cbor": "9fff",
```

Comment:
@AngelCastilloB . Please review this change. I had to update this as it was failing after the CBOR list fix. I assumed this CBOR was also wrong.

Reply:
This change is correct @mirceahasegan ✅